src/main/java/org/icgc_argo/workflow/search/model/graphql/Run.java

```diff
@@ -0,0 +1,59 @@
+package org.icgc_argo.workflow.search.model.graphql;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Map;
+import lombok.*;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Run {
+
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+
+  private String runId;
+
+  private String runName;
+
+  private String startTime;
+
+  private String state;
+
+  private Boolean success;
+
+  private Integer exitStatus;
+
+  private String errorReport;
+
+  private String duration;
+
+  private String completeTime;
+
+  private String commandLine;
+
+  private EngineParameters engineParameters;
+
+  private Object parameters;
+
+  @Data
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  public static final class EngineParameters {
```

Comment:
this is preference but instead of manually annotating with `@JsonPropery` for each of the fields to make them render in snake_case you can annotate this class with `@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)` ... I do this in `workflow-management` for this exact model

Reply:
You are correct. This is an artifact left over of how I implemented it piecemeal. Fixed.